docs/sources/administration/data-source-management/team_ldap.md

```diff
@@ -0,0 +1,130 @@
+
+#### Best practices
+For configuring new rules, or viewing the rules for the teams we recommend testing the rules in the Loki Explore view. This will allow you to see the logs that would be returned for the rule.
+
+
+#### Scenarios
+**Scenario 1: One rule setup for a Team**
+We have two teams, Team A and Team B. Loki access is setup with `Editor`, `Viewer` roles to have `Query` permission.
+
+Team A has a rule `namespace="auth"` configured. 
+
+Team B does not have a rule configured for it. 
+
+**Log Access**
+Team A will have access to logs that match `namespace="auth"`.
+
+Team B will have access to all logs (due to the query permission for the user).
+
+**Scenario 2: Multiple rules setup for two teams**
+We have two teams, Team A and Team B. Loki access is setup with `Admin` roles having `Query` permission.
+
+Team A has rule `namespace="auth", namespace="security"` configured. 
+
+Team B has rule `namespace!="auth", namespace="security"` configured. 
+
+**Log Access**
+Team A will have access to logs that match `namespace="auth"` `OR` `namespace=security`.
+
+Team B will have access to logs that match `namespace!="auth"` `OR` `namespace=security`.
+
+Any admin still have access to query all logs (due to the query permission for the user).
+
+**Scenario 3: Two or more rules configured for a team**
+We have three teams, Team A, Team B and Team C. Team C is a new team and has no rules configured for it. Loki access is setup with `Editor` role having `Query` permission.
+
+Team A has a rule `namespace="auth"` configured.
+
+Team B have two rules `namespace="auth"` and another rule `namespace="security"`. 
+
+Team C have no rules. Team C has a viewer and a editor.
+
+**Log Access**
+Team A will have access to logs that match `namespace="auth"`.
+
+Team B, would be able to query all logs that match either `namespace="auth"` OR `namespace="security"`.
+
+Team C's `Editors`, will have access to query all logs.
+
+> **NOTE:** There will be a warning for all of the roles that have access to query all logs, if there are any TeamLBAC rules configured for the data source.
+
+## Setting up Team LBAC rules
+
+To be able to use Team LBAC rules, you need to enable the feature toggle `teamHTTPHeaders` on your Grafana instance. Contact support to enable the feature toggle for you.
+
+### Prerequisites
+
+### Required permissions
+
+Ensure you have administrative access to Grafana Cloud (GCom) and the necessary permissions to configure data sources and access policies. You will need to have access to the teams you want to configure rules for.
+
+To configure Team LBAC rules, you need to have admin permissions for the data source and edit permissions on the teams you want to configure rules for.
+
+### Steps to Configure Team LBAC Rules for a team
+
+1. Navigate to your Loki datasource
+1. Navigate to the permissions tab
+   - Here, you'll find the Team LBAC rules section.
+1. Add a Team LBAC Rule
+   - If a relevant team doesn't exist, create one.
```

Comment:
It's pretty confusing, why do you need to create rules for non-existing team?

Reply:
yea, let's just skip this section and make it so that we have a way to indicate to the users that a team is needed